MonoGame.Framework/Graphics/Shader/Shader.cs

```diff
@@ -113,7 +113,7 @@ internal Shader(GraphicsDevice device, BinaryReader reader)
                 Attributes[a].location = reader.ReadInt16();
             }
 
-            PlatformConstruct(isVertexShader, shaderBytecode);
+            PlatformConstruct(Stage, shaderBytecode);

```

Comment:
This was a convenient change which seemed good for future support of other shader stages.